src/components/utils/api.js

```diff
@@ -16,6 +16,15 @@ export const getArticles = async () => {
   }
 };
 
+export const getArticleById = async (articleId) => {
+  try {
+    const response = await api.get(`/articles/${articleId}`);
+    return response.data.article
+  } catch (error) {
```

Comment:
Do you need to catch this here, only to throw it straight away? 

Need to catch at the end of your chain